main.go

```diff
@@ -163,65 +180,96 @@ func delete(cmd *cobra.Command, args []string) error {
 }
 
 func listDbs(cmd *cobra.Command, args []string) error {
-	cc, err := client.NewClientWithDefaults()
-	if err != nil {
-		return err
+	dbs, err := getDbs()
+	for _, db := range dbs {
+		fmt.Println(db)
 	}
-	dd, err := cc.DataPath()
+	return err
+}
+
+// getDbs: returns a formatted list of available Skate DBs
+func getDbs() ([]string, error) {
+	filepath, err := getFilePath()
 	if err != nil {
-		return err
+		return nil, err
 	}
-	dbs, err := os.ReadDir(filepath.Join(dd, "kv"))
+	entries, err := os.ReadDir(filepath)
 	if err != nil {
-		return err
+		return nil, err
 	}
-	for _, d := range dbs {
-		if d.IsDir() {
-			fmt.Println("@" + d.Name())
+	var out []string
+	for _, e := range entries {
+		if e.IsDir() {
+			out = append(out, "@"+e.Name())
 		}
 	}
-	return nil
+	return out, nil
 }
 
-func deleteDb(cmd *cobra.Command, args []string) error {
-	n, err := nameFromArgs(args)
-	if err != nil {
-		return err
-	}
+// getFilePath: get the file path to the skate databases.
+func getFilePath(args ...string) (string, error) {
 	cc, err := client.NewClientWithDefaults()
 	if err != nil {
-		return err
+		return "", err
 	}
 	dd, err := cc.DataPath()
 	if err != nil {
-		return err
+		return "", err
 	}
-	dbs, err := os.ReadDir(filepath.Join(dd, "kv"))
+	return filepath.Join(append([]string{dd, "kv"}, args...)...), err
+}
+
+// deleteDb: delete a Skate database.
+func deleteDb(cmd *cobra.Command, args []string) error {
+	dbs, err := getDbs()
 	if err != nil {
 		return err
 	}
+	path, err := findDb(args[0], dbs)
+	var notFoundErr suggestionNotFoundErr
+	if errors.As(err, &notFoundErr) {
+		fmt.Printf("%q does not exist, %s\n", args[0], err.Error())
+		os.Exit(1)
+	}
```

Comment:
```suggestion
	var notFoundErr suggestionNotFoundErr
	if errors.As(err, &notFoundErr) {
		fmt.Printf("%q does not exist, %s\n", args[0], err.Error())
		os.Exit(1)
	}
	if err != nil {
	  fmt.Printf("unexpected error: %s", err.Error())
	  os.Exit(1)
	}
```

otherwise if some other error happens, it'll not be handled

Reply:
Oh great catch! Thank you :)